src/modules/appointment/models/appointment.entity.ts

```diff
@@ -88,6 +97,20 @@ export class AppointmentEntity {
   })
   scheduledAt: string;
 
+  // @Column({
+  //   name: 'treatment_category_id',
+  //   type: 'int',
+  //   nullable: true,
+  // })
+  // treatmentCategoryId: number;
+
```

Comment:
same here?

Reply:
@imadiaz I have already deleted the commented code